.github/workflows/deploy-gh-pages.yaml

```diff
@@ -0,0 +1,94 @@
+# This workflow will deploy the docs as static content to GitHub Pages
+
+name: ☄️ Deploy Documentation
+
+on:
+  workflow_dispatch:
+    inputs:
+      basePath:
+        description: |
+          The base path to deploy the docs to. Default is "oxygen-ui" (https://wso2.github.io/oxygen-ui/).
```

Comment:
Should there be a trailing slash here?

Reply:
Trailing slashes are not allowed by `Next`.
An exception is thrown. I guess they handle the slashes internally.